duktape/src/main/java/com/squareup/duktape/Duktape.java

```diff
@@ -95,6 +97,33 @@ public String evaluate(String script) {
   }
 
   /**
+   * Creates a proxy to a global JavaScript object called {@code name} that implements {@code type}.
+   * {@code type} defines the interface implemented in JavaScript that will be accessible to Java.
+   * {@code type} must be an interface that does not extend any other interfaces, and cannot define
+   * any overloaded methods.
+   * <p>
+   * Methods of the interface may return {@code void} or any of the following supported argument
+   * types: {@code boolean}, {@link Boolean}, {@code int}, {@link Integer}, {@code double},
+   * {@link Double}, {@link String}.
+   * </p>
```

Comment:
also, if you put the `<p>` on the same line as `Methods`, then IntelliJ is able to reformat automatically. That’s what we do in our other HTML Javadocs.